packages/client/src/runtime/RequestHandler.ts

```diff
@@ -154,37 +158,42 @@ export class RequestHandler {
         return { data: unpackResult, elapsed }
       }
       return unpackResult
-    } catch (e: any) {
-      debug(e)
-      let message = e.message
-      if (callsite) {
-        const { stack } = printStack({
-          callsite,
-          originalMethod: clientMethod,
-          onUs: e.isPanic,
-          showColors,
-        })
-        message = `${stack}\n  ${e.message}`
-      }
+    } catch (error) {
+      this.handleRequestError({ error, clientMethod, callsite })
```

Comment:
Nice, thanks for the cleanup.

Reply:
See [this commit](https://github.com/prisma/prisma/pull/13811/commits/7ad50cb785f124a156628363b6c13c2ce5ae9bd6) and [this comment](https://github.com/prisma/prisma/pull/13811#issuecomment-1159798372) for details.

Reply:
Ah perfect. Thanks!